src/api/libcellml/variable.h

```diff
@@ -140,28 +140,36 @@ class LIBCELLML_EXPORT Variable: public NamedEntity
 
     /**
      * @brief Get the initial value for this variable.
-     * Get the real number initial value for this variable.
+     * Get the string corresponding to the initial value for this variable.
      * @sa setInitialValue
      */
     std::string getInitialValue() const;
 
     /**
      * @brief Set the interface type for this variable.
+     * Set the interface type for this variable using a string.
+     * @sa getInterfaceType
+     * @param interfaceType The string interface type to set for this variable.
+     */
+    void setInterfaceType(const std::string &interfaceType);
+
+    /**
+     * @brief Set the interface type for this variable.
      * Set the interface type for this variable from the available
-     * options in the INTERFACE_TYPES enum.
+     * options in the InterfaceTypes enum class.
+     *
+     * @overload
      * @sa getInterfaceType
-     * @param interfaceType The interface type to set for this variable.
+     * @param interfaceType The enum interface type to set for this variable.
      */
-    void setInterfaceType(INTERFACE_TYPES interfaceType);
+    void setInterfaceType(InterfaceTypes interfaceType);
```

Comment:
Nitpicking: I really find it odd to have a type called ```InterfaceTypes``` (rather than ```InterfaceType```, i.e. singular) when the argument refers to one and only interface type... (Same goes with ```Formats``` rather than ```Format```.)